daisybell/__main__.py

```diff
@@ -7,7 +7,7 @@
 from tabulate import tabulate
 import pandas as pd
 
-from daisybell import scan
+from daisybell.daisybell import scan
```

Comment:
Probably should be at least imported in __init__.py so it's 

 `from daisybell import scan`

Reply:
Done